test/fork/switchImplementation/snapshots/MiltonSnapshot.sol

```diff
@@ -66,8 +68,10 @@ contract MiltonSnapshot is Script, Test {
         miltonMaxSwapCollateralAmount = milton
         .getMaxSwapCollateralAmount();
         miltonMaxLpUtilizationRate = milton.getMaxLpUtilizationRate();
-        miltonMaxLpUtilizationPerLegRate = milton
-        .getMaxLpUtilizationPerLegRate();
+//        miltonMaxLpUtilizationRatePayFixed = milton
+//        .getMaxLpUtilizationRatePayFixed(); TODO revert
```

Comment:
do we have tasks for that?

Reply:
after deployment we have to revert it, so it will be useful